### DISCUSSION TITLE
Bun error on request

### DISCUSSION THREAD

**Discussion by chrisvander** [vikejs/vike#1594] [Discussion]
Hi, I'm testing out Bun to see how it fits in my project's workflow and whether it brings some additional speed to development. The server appears to set itself up okay, but Vike throws an error on HTTP request:
```
11:26:49 AM [vike][request(1)] HTTP request: /studies
583 |     if (aNeedle[aLineName] <= 0) {
584 |       throw new TypeError('Line must be greater than or equal to 1, got '
585 |                           + aNeedle[aLineName]);
586 |     }
587 |     if (aNeedle[aColumnName] < 0) {
588 |       throw new TypeError('Column must be greater than or equal to 0, got '
                  ^
TypeError: Column must be greater than or equal to 0, got -1
```

Vike 0.4.160, Bun 1.1.0

**Comment by brillout** [comment]
Since it's most likely an issue with Bun, I'm reluctant to implement workarounds for Bun bugs. That said, PR welcome if it there is a workaround that doesn't complexify Vike.

**Reply by brillout** [reply]
Actually, `aNeedle` isn't Vike code, so I don't think there is any relation to Vike.

**Reply by chrisvander** [reply]
Good point, related indicates that this is an issue with stack traces. Maybe there's a different issue. https://github.com/remix-run/remix/discussions/8482
